kotkt.nl/content/ocw/_index.nl.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Open Cultuur Week"
+---
+
+{{< articleimage
+    url="/images/ocw2020.png" 
+>}}
+
+Van 9 september tot en met 11 september organiseren we een paar activiteiten weer alle studenten welkom zijn om mee te doen.
+Deze activiteiten zijn een deel van de [Open Cultuur Week](https://studentencultuur.nl/OCW/) waar alle Studenten Cultuurverenigingen van Eindhoven de deuren open zetten voor nieuwe leden.
+
+{{< aside >}}
+  {{< banner rotate="-5" padding="2px">}}
+      Vanwege COVID-19 is het verplicht om je aan te melden om mee te doen met een van de activiteiten.
+      <a href="https://forms.gle/f1kBn4c8SWdf1GBQ9">Meld je hier aan</a>
+   {{< /banner >}}
+{{</ aside >}}
+
+Zoals iedereen moeten we rekening houden met COVID-19 en hebben we regels opgesteld om ervoor te zorgen dat onze activiteiten veilig plaats kunnen vinden.
+Je kan [hier](https://beta.kotkt.nl/nextcloud/s/x5p6TNtqqzLHiqp/download) onze regels vinden, we raden aan ze door te lezen voordat je langskomt.
```

Comment:
seems a bit odd to put the link in an aside, but also in here. what's up with that?

Reply:
You don't see asides on mobile phones, so if people are using their fondleslab we want them to see the link as well. Because sign up is important I want to repeat the link multiple times.